doc/source/raysgd/v2/user_guide.rst

```diff
@@ -294,14 +540,41 @@ Here is an example:
 
     import torch
     import torchmetrics
-    import ray
+    from ray.util.sgd.v2 import Trainer
 
     def train_func(config):
         preds = torch.randn(10, 5).softmax(dim=-1)
         target = torch.randint(5, (10,))
+        # In a real use-case this would be passed into sgd.report()
```

Comment:
I think we should show the full real use-case using `sgd.report()` in this example itself.

Reply:
Updated!